### DISCUSSION TITLE
这个型号，能识别不能控制

### DISCUSSION THREAD

**Discussion by star2k** [al-one/hass-xiaomi-miot#1983] [General]
![image](https://github.com/user-attachments/assets/5987a29f-f374-4e20-9867-03f7edadea40)!

**Comment by star2k** [comment]
除了第一个，下面开关按钮，一个都点不了，第一个也是无效开关，希望能早日匹配控制

**Comment by al-one** [comment]
设备问题[按此格式](https://github.com/al-one/hass-xiaomi-miot/issues/new?template=device-issue.yml)提交
